packages/nuxt/src/app/composables/manifest.ts

```diff
@@ -24,9 +24,7 @@ function fetchManifest () {
   if (!isAppManifestEnabled) {
     throw new Error('[nuxt] app manifest should be enabled with `experimental.appManifest`')
   }
-  manifest = $fetch<NuxtAppManifest>(buildAssetsURL(`builds/meta/${useRuntimeConfig().app.buildId}.json`), {
-    responseType: 'json',
-  })
+  manifest = fetch(buildAssetsURL(`builds/meta/${useRuntimeConfig().app.buildId}.json`)).then(r => r.json() as Promise<NuxtAppManifest>)
```

Comment:
One immediate downside: There won’t be any retry mechanisms anymore. 

Does it really improve performance or any compatibility ?

Reply:
oh, that's a very good point ...

it slightly improves performance simply by reducing the number of functions called (as `fetch` underlies `ofetch` anyway) - and removes possible edge cases that might impair the initial load

of course, I would prefer to see this 'fixed' in `ofetch` if there is in fact any bug (such as passing `method: undefined` as the issue suggests) ...

Reply:
yes, that is my impression